packages/koa-content-security-policy/index.js

```diff
@@ -0,0 +1,61 @@
+const uniqid = require('uniqid');
+
+const configProperties = ['styleSrc', 'frameAncestors', 'fontSrc', 'scriptSrc'];
+const mergeAndSanitizeConfigs = (...args) =>
+  args.reduce(
+    (prevConfig, config) =>
+      configProperties.reduce(
+        (resultConfig, property) => ({
+          ...resultConfig,
```

Comment:
this is an anti pattern -> https://www.richsnapp.com/article/2019/06-09-reduce-spread-anti-pattern

```js
configProperties.reduce(
        (resultConfig, property) => {
          resultConfig[property] = [...(prevConfig[property] || []), ...(config[property] || [])];
          return resultConfig;
        },
        {}
      )
```